### PR TITLE
Add roadmap - 1.0.0 and 1.1.0 releases

### PR DESCRIPTION
**Pull request by xtuc**
Please let me know if you have anything to add in here.

PRs https://github.com/webpack/webpack/pull/6531 https://github.com/webpack/webpack/pull/6533 for ref.

### CODE REVIEW

**Comment by xtuc** [comment]
I could add better error handling in case of a malformed wasm binary. I saw Chrome showing the WAST equivalent and pointing to the error.

File: docs/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Roadmap
+
+## Version 1.0.0
+
+There are a few PRs against Webpack to use directly or indirectly this project.
```
Comment (ColinEberhardt) [inline docs/roadmap.md]: Can you link to these PRs? it would be good to know how this project is potentially being used.

File: docs/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - [x] WAST - Start the execution from the start function
+    
+## Version 1.1.0
+
+The second important feature for Webpack is [wasm-dce](https://github.com/xtuc/wasm-dce).
+
+It depends on webassembly-interpreter, and thus the following work is todo:
+- Improve WASM support
+  - [ ] WASM code generation (https://github.com/xtuc/js-webassembly-interpreter/issues/68)
+  - [ ] Transform our current WAST test suite to WASM and relaunch the same tests.
```
Comment (ColinEberhardt) [inline docs/roadmap.md]: I think the overall test approach could do with being documented. There are extensive tests for:

1. The WAT parser
2. The interpreter
3. And a growing set of tests based on the spec test suite

The testing around WASM is less thorough, and could potentially make use of all of the above.

File: docs/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Roadmap
```
Comment (ColinEberhardt) [inline docs/roadmap.md]: Is it worth adding https://github.com/xtuc/js-webassembly-interpreter/issues/152 to the roadmap? I'd worry that without this fix this project is going to need to make a breaking change just to rename parts of the API

**Comment by ColinEberhardt** [comment]
On a related note, is there a plan to enhance wasm-dce to remove non-imported unused functions? similar to wasm-gc.

File: docs/roadmap.md (hunk 3 above)
Comment (xtuc) [reply docs/roadmap.md]: Yes good point, i'll add it.

File: docs/roadmap.md (hunk 2 above)
Comment (xtuc) [reply docs/roadmap.md]: I can reword it like that: 
> use our existing WAT and interpreter tests against our WASM implementation.

**Comment by xtuc** [comment]
> On a related note, is there a plan to enhance wasm-dce to remove non-imported unused functions? similar to wasm-gc.

I'm not sure because usually the compilers (C++/Rust) will remove them before we get the binary.

**Comment by xtuc** [comment]
I'm merging this now, feel free to edit it if you want to add/update items.

**Pull request merged**